app/mailers/acceptance.rb

```diff
@@ -0,0 +1,6 @@
+class Acceptance < ApplicationMailer
```

Comment:
nit: It might be a good idea to name this `AcceptanceMailer` so that when this is used it's clear what it's doing.